public/index.html

```diff
@@ -1,20 +1,18 @@
 <!DOCTYPE html>
 <html lang="">
-  <head>
+<head>
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <meta content="IE=edge" http-equiv="X-UA-Compatible">
+    <meta content="width=device-width,initial-scale=1.0" name="viewport">
+    <link href="<%= BASE_URL %>favicon.ico" rel="icon">
     <title><%= htmlWebpackPlugin.options.title %></title>
-
-    <meta name="viewport" content="initial-scale=1, viewport-fit=cover, width=device-width"></meta>
-    <meta name="apple-mobile-web-app-capable" content="yes"></meta>
-    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent"></meta>
```

Comment:
Why remove this? It made the side bars re-appear on iOS devices

Reply:
Because it's now added automaticly by Wepack on build according to `vue.config.js`